### PR TITLE
Include `killed` boolean in command result

### PR DESCRIPTION
**Pull request by Roaders**
The exit code of each command is already included in the result but the usefulness of this is limited if concurrently is run with `killOthers`. When parsing the result you do not know if a non-0 exit code is because the command failed or if it was killed by concurrently.

Adding this option allows you to print a message at the end identifying exactly which commands caused the initial failure.

### CODE REVIEW

**Comment by Roaders** [comment]
Any chance we can get this PR considered?

**Review by gustavohenke — CHANGES_REQUESTED** [review]
Hey! Can we also get some test for this, please?

**Comment by Roaders** [comment]
I was going to write tests but tests were failing before I started so it's hard to know where to start.

**Comment by Roaders** [comment]
As far as I can tell there are no existing tests that test the exit condition of multiple commands in the `close` stream. I am trying to find one to base my tests on.

**Comment by Roaders** [comment]
OK, I figured it out. Been so long since I did this I had forgotten the structure of the project.

File: src/command.spec.js [hunk 1]
```diff
@@ -164,4 +167,17 @@ describe('#kill()', () => {
 
         expect(killProcess).not.toHaveBeenCalled();
     });
+
+    it('should mark the command as killed', done => {
```
Comment (gustavohenke) [inline src/command.spec.js:171]:
```suggestion
    it('marks the command as killed', done => {
```

**Review by gustavohenke — APPROVED** [review]
Thanks for the side fixes!

**Comment by Roaders** [comment]
Do I have to do anything to merge this? I assume you'll merge it at some point.
Thanks.

**Comment by Roaders** [comment]
Any idea when this will be released?

**Comment by gustavohenke** [comment]
Heya. Just released it in v6.2.0.

**Comment by Roaders** [comment]
brilliant, thanks

**Pull request merged**